docs/admin.rst

```diff
@@ -7,12 +7,23 @@ This chapter describes maintenance tasks for the administrator of a GHC instance
 There is a separate :ref:`userguide` that provides guidance to the end-user to
 configure the actual Resource healthchecks.
 
-Each of the sections below is geared at a specific administrative task area.
+Each of the sections below is geared at a specific administrative task area. We
+assume you have installed GeoHealthCheck with `pip install`, and that you have
+acces to the command line interface `geohc`. You can get help from the cli via ::
+
+    geohc --help
+
+This will list all available commands. To get help with a specific command, run
+`geohc <<command>> --help`, e.g. ::
+
+    geohc db-adduser --help
```

Comment:
Suggest using subcommands like `ghc db adduser`, `ghc db create`

Reply:
Done (commit: 040aa0a). How about other subcommands for `run`: `run app`, `run tests`, `run healthchecks`, `run daemon` and `create`: `hash`, `wsgi`, `secret-key`, ... or would that be over the top?

Reply:
Great idea. Perhaps cover in another PR / discuss to design our subcommands.

Reply:
I opened issue #311 to discuss the design of the subcommands.